core/testsuite/src/test/java/com/blazebit/persistence/testsuite/JsonGetAndSetTest.java

```diff
@@ -113,6 +112,46 @@ public void testJsonGet() throws JsonProcessingException {
         assertEquals(2, arrayRootResult.get(0).get(2));
     }
 
+    @Test
+    @Category({ NoH2.class, NoSQLite.class, NoFirebird.class, NoOracle.class })
+    public void testJsonGetWithJsonPath() throws JsonProcessingException {
+        List<Tuple> objectRootResult = cbf.create(em, Tuple.class).from(JsonDocument.class, "d")
+            .select("d.content")
+            .select("json_get(d.content, '$.K1[??].K3', :array_index_2)")
+            .select("json_get(d.content, '$.K1[??]', :array_index_0)")
+            .select("json_get(d.content, '$.??[0]', :k1)")
+            .select("json_get(d.content, '$.??[??]', :k1, :array_index_0)")
+            .select("json_get(d.content, '$.K1[0]')")
```

Comment:
Please add that as separate test method, so that it can run also on Oracle.

Reply:
Done